src/Types.ts

```diff
@@ -44,6 +44,8 @@ export type EditorState = {
   _cachedSelectedText?: SelectedText;
   _pushTextToEditor?: string;
 
+  _pushContentToEditor?: string;
```

Comment:
This is a major change that you should find in the librarySlice file. 

Right now you are saving the full quill object to the store, but you should never store non-serializable objects to a store. While this does not really affect us (you don't want to do this due to server side rendering and hydration), it is smart to follow the correct patterns. 

I have modified the content to use this instead.